Tests/PenguinStructuresTests/ArrayStorageTests.swift

```diff
@@ -141,7 +236,11 @@ class ArrayStorageTests: XCTestCase {
     ("test_create", test_create),
     ("test_append", test_append),
     ("test_typeErasedAppend", test_typeErasedAppend),
+    ("test_appending", test_appending),
+    ("test_typeErasedAppending", test_typeErasedAppending),
+    ("test_replacementStorage", test_replacementStorage),
```

Comment:
Nit: it's nice to keep these in the same order as the tests are in the source file.

Reply:
That seems like a bunch of busywork for little benefit, but I will do it if you think it's a good policy.  Should we also alphabetize them? (0.5 😉 )

Reply:
Done